runtime/doc/dev_tools.txt

```diff
@@ -14,32 +14,40 @@ itself. See also |debug.txt| for advice that applies to Vim.
 ==============================================================================
 Backtraces                                            *dev-tools-backtrace*
 
-LINUX ~
+LINUX
 
-Core dumps are disabled by default on Ubuntu
-https://stackoverflow.com/a/18368068, CentOS and others. To enable core dumps:
+Core dumps are disabled by default on Ubuntu, CentOS and others.
+To enable core dumps:
 >bash
     ulimit -c unlimited
 <
 On systemd-based systems getting a backtrace is as easy as:
 >bash
     coredumpctl -1 gdb
 <
-It's an optional tool, so you may need to install it:
+`corecumpctl` is an optional tool, so you may need to install it:
 >bash
     sudo apt install systemd-coredump
 <
 
-The full backtrace is most useful, send us the `bt.txt` file:
```

Comment:
I think I like the old formulation more, there's less risk of misunderstanding. "Send us bt.txt" vs "send us backtrace information".

Reply:
I don't mind with either, so I switched back to the previous one. :)